reana_workflow_controller/app.py

```diff
@@ -42,6 +42,34 @@
 }
 
 
+@app.route('/workflows', methods=['GET'])
+def get_workflows():
+    """Get all workflows.
+    
+    .. http:get:: /workflows
+        Returns a JSON list with all the workflows.
+        NOTE: Currently just a mock that can used e.g. to quickly
+        check if reana-workflow-controller is running.
+        **Request**:
+        .. sourcecode:: http
+            GET /workflows HTTP/1.1
+            Content-Type: application/json
+            Host: localhost:5000
+        :reqheader Content-Type: application/json
+        **Responses**:
+        .. sourcecode:: http
+            HTTP/1.1 200 OK
+            Content-Length: 22
+            Content-Type: application/json
+            {
+              "workflows": {}
+            }
+        :resheader Content-Type: application/json
+        :statuscode 200: no error - the list has been returned.
+    """
```

Comment:
Remember that you should reference this new endpoint in https://raw.githubusercontent.com/reanahub/reana-workflow-controller/master/docs/restapi.rst otherwise it doesn't get rendered